### DISCUSSION TITLE
NVIDIA FRUC

### DISCUSSION THREAD

**Discussion by ghost** [hooke007/mpv_PlayKit#239] [mpv 通用]

**Comment by hooke007** [comment]
这你要到上游的具体实现仓库去询问。
不算新的fruc，只看NVOF的话在svp中很早就有实验了，效果很差。

**Reply by hooke007** [reply]
局部受 https://github.com/hooke007/MPV_lazy/wiki/3_K7sfunc#svp_pro 支持
